lib/smart_app_launch/smart_stu2_suite.rb

```diff
@@ -210,14 +211,46 @@ class SMARTSTU2Suite < Inferno::TestSuite
     group do
       title 'Token Introspection'
       id :smart_token_introspection
-
-      input_instructions <<~INSTRUCTIONS
-        TODO: Instructions for token introspection go here!
-      INSTRUCTIONS
+      description %(
+        # Background
+
+        OAuth 2.0 Token introspection, as described in [RFC-7662](https://datatracker.ietf.org/doc/html/rfc7662), allows
+        an authorized resource server to query an OAuth 2.0 authorization server for metadata on a token.  The
+        [SMART App Launch STU 2.1 Implementation Guide Section on Token Introspection](https://hl7.org/fhir/smart-app-launch/token-introspection.html)
+        states that "SMART on FHIR EHRs SHOULD support token introspection, which allows a broader ecosystem of resource servers
+        to leverage authorization decisions managed by a single authorization server."
+  
```

Comment:
```suggestion
 
```

Very nitpicky, but a single space here caused the markdown to completely break and show "# Background" etc instead of the parsed markdown.  This is because Markdown is sensitive to lead spaces, and our interface preprocesses the block of text before passing it to the markdown component and attempts to intelligently move the whole block 'left' the corrent number of spots.

Reply:
Thank you!! I couldn't figure out what was causing the formatting issue, I'll remember a leading space in the future.